config/install/field.field.paragraph.gated_content.field_prgf_link.yml

```diff
@@ -0,0 +1,27 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.paragraph.field_prgf_link
+    - paragraphs.paragraphs_type.gated_content
+  module:
+    - datalayer
+    - link
+third_party_settings:
+  datalayer:
+    expose: 0
+    label: field_prgf_link
+id: paragraph.gated_content.field_prgf_link
+field_name: field_prgf_link
+entity_type: paragraph
+bundle: gated_content
+label: 'Headline Link'
+description: ''
```

Comment:
would be great to have a description for content managers
https://github.com/ymcatwincities/openy/wiki/Code-Review-Quality-Best-Practices#generic-rules